src/openvic-simulation/map/ProvinceInstance.cpp

```diff
@@ -185,6 +191,73 @@ void ProvinceInstance::_update_pops(DefineManager const& define_manager) {
 	}
 }
 
+void ProvinceInstance::update_modifier_sum(Date today, StaticModifierCache const& static_modifier_cache) {
```

Comment:
Shouldn't the province update the owner?
Now we're looping through all provinces twice per day tick. Once via map and once via country.

Reply:
I handling the province-to-owner contributions on the province side, however there's an issue: we'd still need to clear each country's modifier sum before adding any province contributions. Still, that may be worth it to avoid a loop over every owned province.